packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.tsx

```diff
@@ -97,7 +98,7 @@ export class HoverCard extends BaseComponent<IHoverCardProps, IHoverCardState> {
             {...getNativeProps(this.props, divProperties)}
             id={hoverCardId}
             trapFocus={!!this.props.trapFocus}
-            firstFocus={openMode === OpenCardMode.hotKey || openMode === OpenCardMode.hover}
+            firstFocus={!!this.props.defaultFirstFocus || openMode === OpenCardMode.hotKey}
```

Comment:
We don't want to set focus in hovercard when it is opened by hover behavior. So make it possible to disable first focus.

Reply:
I feel like the focus logic here is inconsistent with other things. @atneik @chaofeng2014 it would be helpful to think about this generally; Callout, TeachingBubble, etc are similar components. What should be the correct design here?

I believe those have `setInitialFocus` implying the default behavior does NOT move focus into the card. Perhaps HoverCard should be following this model for consistency?

Reply:
We should rename it to `setInitialFocus` for consistency. 

The hover action shouldn't take away your focus by default. If there is an implicit action (keydown/click) then it makes sense to set the focus.

I see HoverCard and Tooltip having same logic.